test/index.test.ts

```diff
@@ -0,0 +1,82 @@
+import { expect } from "chai";
+import { Chart } from "chart.js";
+
+import ChartUtils from "../src/index.ts";
+
+describe("Testing ChartUtils", () => {
+  it("Testing afterDatasetsUpdate", () => {
+
+    const _meta = {
```

Comment:
would propose removing the underscore, this is just another const, nothing special :)